tests/cloudpickle_test.py

```diff
@@ -655,11 +655,16 @@ class ConcreteClass(AbstractClass):
             def foo(self):
                 return 'it works!'
 
+        AbstractClass.register(tuple)
+
         depickled_base = pickle_depickle(AbstractClass, protocol=self.protocol)
```

Comment:
This introduces a side effect in the test. I don't think it's possible to unregister tuple. Cloud you please instead use a new locally defined class, e.g.:

```python

class SomeOtherClass(object):
    pass

AbstractClass.register(SomeOtherClass)
...

```

Reply:
Actually, it should also be possible to do:

```python
AbstractClass.register(tuple)
try:
   ...
finally:
    AbstractClass._abc_registry_clear()
```

do as you wish.

Reply:
We would also need to clean the registry of `depickled_base`. Better not register `tuple` at all.